### PR TITLE
[EAP7-1795] EAP7 applications have to communicate with EAP8 apps succesfully

### PR DESCRIPTION
**Pull request by ropalka**
https://issues.redhat.com/browse/EAP7-1795

### CODE REVIEW

File: backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc [hunk 1]
```diff
@@ -0,0 +1,219 @@
+= [EAP7-1795] EAP6 and EAP7 applications have to communicate with EAP8 apps successfully and vice versa
```
[the remark this answers is not in the document]
Comment (bstansberry) [reply backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc:1]: @ropalka It's still very focused on EAP7.

**Comment by ropalka** [comment]
How about this proposal @bstansberry ?

**Comment by bstansberry** [comment]
@ropalka This seems fine to me, except please change the commit message, PR title, PR description JIRA link and analysis file name to reference WFLY-16296, not EAP7-1795. The EAP7 project is used by Red Hat for it's internal product planning work, and we Red Hatters are familiar with it and deal with it in our work, but it's not a community JIRA project and is not visible to community.

**Comment by darranl** [comment]
@ropalka can you please clean up the EAP7 references on this one so I can get it merged?

**Comment by ropalka** [comment]
Done @bstansberry @darranl - see https://github.com/wildfly/wildfly-proposals/pull/526 . Closing this one.

**Pull request closed without merging**